src/ch/blinkenlights/android/vanilla/SortableAdapter.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2017 Adrian Ulrich <adrian@blinkenlights.ch>
```

Comment:
Most of the code is copy&pasted. And likely it will be modified in the future. Don't like this copyleft stuff and git blame is enough.